### ISSUE TITLE
Update Serbo-Croatian translations

### ISSUE THREAD

**Issue by MladenPuzic** [squidfunk/mkdocs-material#5861]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "sh",
  "action.edit": "Ažuriraj stranicu",
  "action.skip": "Idi na tekst",
  "action.view": "Pogledaj izvorni kod ove stranice",
  "announce.dismiss": "Nemoj mi ponovo pokazati ovo",
  "blog.archive": "Arhiva",
  "blog.categories": "Kategorije",
  "blog.categories.in": "u",
  "blog.continue": "Nastavi sa čitanjem",
  "blog.draft": "Draft ⬅️",
  "blog.index": "Back to index ⬅️",
  "blog.meta": "Metapodaci",
  "blog.references": "Povezani linkovi",
  "clipboard.copy": "Kopiraj u klipbord",
  "clipboard.copied": "Iskopirano u klipbord",
  "consent.accept": "Prihvati",
  "consent.manage": "Promeni podešavanja",
  "consent.reject": "Odbij",
  "footer": "Podnožje",
  "footer.next": "Sledeće",
  "footer.previous": "Prethodno",
  "header": "Zaglavlje",
  "meta.comments": "Komentari",
  "meta.source": "Izvor",
  "nav": "Navigacija",
  "readtime.one": "1 minut čitanja",
  "readtime.other": "# minuta čitanja",
  "rss.created": "RSS feed ⬅️",
  "rss.updated": "RSS feed of updated content ⬅️",
  "search": "Pretraga",
  "search.placeholder": "Pretraga",
  "search.share": "Deljenje",
  "search.reset": "Očisti",
  "search.result.initializer": "Inicijalizujem pretragu",
  "search.result.placeholder": "Unesite pojam pretrage",
  "search.result.none": "Ništa nije pronađeno",
  "search.result.one": "1 rezultat pretrage",
  "search.result.other": "# rezultata pretrage",
  "search.result.more.one": "još 1 na ovoj strani",
  "search.result.more.other": "još # na ovoj strani",
  "search.result.term.missing": "Nedostaje",
  "select.language": "Izaberi jezik",
  "select.version": "Izaberi verziju",
  "source": "Idi u repozitorijum",
  "source.file.contributors": "Contributors ⬅️",
  "source.file.date.created": "Kreiran",
  "source.file.date.updated": "Ažuriran",
  "tabs": "Tabovi",
  "toc": "Sadržaj",
  "top": "Nazad na vrh"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 688c9de4c.

**Comment by squidfunk** [comment]
Released as part of [9.2.0](https://github.com/squidfunk/mkdocs-material/releases/tag/9.2.0)!
